packages/react/src/components/Rating/Rating.types.ts

```diff
@@ -110,11 +110,16 @@ export interface IRatingProps extends React.HTMLAttributes<HTMLDivElement>, Reac
    */
   readOnly?: boolean;
 
-  /*
-   * Optional callback to set the aria-label for rating control.
+  /**
+   * Optional callback to set the aria-label for rating control in readOnly mode
```

Comment:
Changing the behavior to not call this by default is going to break anyone who was relying on this prop to provide the label in the past. We probably need to find another way to handle this.

Reply:
based on the code (below), I think this comment is incorrect. It appears to to work even in editable mode, except if `ariaLabel` is provided.

```ts
const readOnlyAriaLabel = getAriaLabel?.(displayRating, max);
const normalModeAriaLabel = ariaLabel ? ariaLabel : readOnlyAriaLabel;
```